docs/workflows/copy.rst

```diff
@@ -1,43 +1,190 @@
+recipe section below for more details on how to do this.
+
+
+Recipes
+_______
+
+These are examples of how the RPM copy API should be used. This code isn't intended to be runnable
+as-is, but rather as a template for how the calls should be constructed.
+
+Create a new repository version in "dest_repo" containing the two "content" units specified by href,
+which are present in the "source_repo_version".
 
 .. code:: json
 
-    {
-        "base_version": "/pulp/api/v3/repositories/rpm/rpm/c8c632eb-cf9a-494f-8e12-b3e651b6e75e/versions/1/",
-        "content_summary": {
-            "added": {},
-            "present": {
-                "rpm.package": {
-                    "count": 1,
-                    "href": "/pulp/api/v3/content/rpm/packages/?repository_version=/pulp/api/v3/repositories/rpm/rpm/c8c632eb-cf9a-494f-8e12-b3e651b6e75e/versions/3/"
-                }
-            },
-            "removed": {
-                "rpm.advisory": {
-                    "count": 1,
-                    "href": "/pulp/api/v3/content/rpm/advisories/?repository_version_removed=/pulp/api/v3/repositories/rpm/rpm/c8c632eb-cf9a-494f-8e12-b3e651b6e75e/versions/3/"
-                }
-            }
-        },
-        "number": 3,
-        "pulp_created": "2019-11-27T16:07:21.561748Z",
-        "pulp_href": "/pulp/api/v3/repositories/rpm/rpm/c8c632eb-cf9a-494f-8e12-b3e651b6e75e/versions/3/"
-    }
+    POST :24817/pulp/api/v3/rpm/copy/ \
+    config:=[ \
+        {"source_repo_version": "$SRC_REPO_VERS_HREF", "dest_repo": "$DEST_REPO_HREF", "content": [$RPM_HREF1, $ADVISORY_HREF1]}, \
+    ] \
+    dependency_solving=False
+
+Create a new repository version in "dest_repo" containing the two "content" units specified by href,
+which are present in the "source_repo_version". Instead of adding them to the content present in
+the latest repository version present in "dest_repo", instead create a new version based upon
+the version numbered "dest_base_version" in "dest_repo". These semantics are similar to how the
+"base_version" parameter is used in the repository modification API.
+
+.. code:: json
+
+    POST :24817/pulp/api/v3/rpm/copy/ \
+    config:=[ \
+        {"source_repo_version": "$SRC_REPO_VERS_HREF", "dest_repo": "$DEST_REPO_HREF", "dest_base_version": "$DEST_BASE_VERSION", "content": [$RPM_HREF1, $ADVISORY_HREF1]}] \
+    dependency_solving=False
+
+Create a new repository version in "dest_repo" containing the two "content" units specified by href,
+as well as all of their RPM and Module dependencies, which are present in the "source_repo_version".
+
+.. code:: json
+
+    POST :24817/pulp/api/v3/rpm/copy/ \
+    config:=[ \
+        {"source_repo_version": "$SRC_REPO_VERS_HREF", "dest_repo": "$DEST_REPO_HREF", "content": [$RPM_HREF1, $ADVISORY_HREF1]}, \
+    ]
+    dependency_solving=True
+
+"Multi-repository-copy", required when any of the repositories involved in the copy are not "dependency closed".
```

Comment:
Explaining this in a satisfactorily simple way is quite difficult, but I'm sure there is still room for improvement.

Reply:
I think you did a great job in this section.  I can't think of a way describing it easier.